backend/module/eCampCore/src/EntityService/ContentNodeService.php

```diff
@@ -90,8 +91,14 @@ protected function createEntity($data): ContentNode {
             $contentNode->setPosition($position + 1);
         }
 
-        /** @var ContentType $contentType */
-        $contentType = $this->findRelatedEntity(ContentType::class, $data, 'contentTypeId');
+        if (isset($data->contentTypeName)) {
+            /** @var EntityRepository $contentTypeRepository */
+            $contentTypeRepository = $this->getServiceUtils()->emGetRepository(ContentType::class);
```

Comment:
Das Repo kriegst du auch direkt aus dem Service:
`$this->getRepository()`

Reply:
Dann bekommt man das Repository von ContentNode.

Reply:
Ok, hab ich wohl vor lauter Content den Node nicht mehr gesehen :-)